dart/pubspec.yaml

```
@@ -13,7 +13,7 @@ dependencies:
   http: ^0.12.0
   meta: ^1.0.0
   stack_trace: ^1.0.0
-  uuid: ^2.0.0
+  uuid: ^3.0.1
```

It would make sense to add a version range like 
```suggestion
  uuid: >=2.0.0 <4.0.0
```
that way this change wouldn't break backwards compatibility.